source/install/install-kubernetes.rst

```diff
@@ -21,22 +21,76 @@ You'll need a `Kubernetes cluster <https://kubernetes.io/docs/setup/>`__ running
 Install the operators
 ---------------------
 
-Operators are installed using ``kubectl``, and each operator is created in its own namespace. You can install and run multiple Mattermost installations in the same cluster using different namespaces.
+Nginx Ingress Controller
+^^^^^^^^^^^^^^^^^^^^^^^^
+For instructions on installation the NGINX ingress controller in your Kubernetes cluster, click `here <https://kubernetes.github.io/ingress-nginx/deploy/>`__.
 
-1. Install NGINX ingress controller by following the instructions `here <https://kubernetes.github.io/ingress-nginx/deploy/>`__.
+Mattermost Operator
+^^^^^^^^^^^^^
 
-2. Install the Mattermost Operator:
+There are multiple options for installation of the Mattermost Kubernetes Operator. We recommend deployment with Helm, but installation instructions via the Kubernetes CLI can also be found below.
+
+With Helm
+^^^^^^^^^
+
+
+1. Install Helm. We recommend Helm v3.4.0 or later. For installation instructions, click `here <See: https://docs.helm.sh/using_helm/#quickstart>`__.
+2. Once Helm is installed and initialized, run the following:
+
+  .. code-block:: sh
+    :class: mm-code-block 
+
+    helm repo add mattermost https://helm.mattermost.com
+
+3. Create a file named ``config.yaml``, then copy and paste the content of `this file <https://github.com/mattermost/mattermost-helm/blob/master/charts/mattermost-operator/values.yaml>`__ into it.
+
+  .. tip::
+      This will be your configuration file for the Mattermost Operator chart. If left as-is, the Mattermost-Operator will utilize the MySQL Operator, and Minio Operator - **these operators are not recommended for production usage**. You an customize your deployment at any time by updating the relevant values in ``config.yaml``.
```

Comment:
We got rid of the other operators with v1.0.0 of the chart so let's go ahead and remove this.